### DISCUSSION TITLE
video recording 10sec repeat every 1 hour

### DISCUSSION THREAD

**Discussion by lyj6668882002** [gopro/labs#360] [Feature Requests]
hello , I want to ask a question
how can I set QR, let hero11 record video 10sec repeat every 1 hour until no battery power?

**Comment by dnewman-gpsw** [comment]
Easy.  

mV!S!10E!3600R

mV - mode Video
!S - Start
!10E - 10 seconds later End
!3600R  - sleep for 1 hour and repeat.

**Reply by lyj6668882002** [reply]
thank you，it worked，thank you very much。
